herbieapp/models/message_models_serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from rest_framework.fields import JSONField
+
+
+class EntityUpdateMessage:
+    def __init__(self, _type, key, version, payload, created, modified, tags):
+        self.tags = tags
+        self.action = 'update'
```

Comment:
The action strings could be defined as constants somewhere.

Reply:
Yes i agree with you. This is from a previous ticket implementation. I will do the refactor